contrib/slurm/README.md

```diff
@@ -0,0 +1,86 @@
+<!--
+This file is part of BenchExec, a framework for reliable benchmarking:
+https://github.com/sosy-lab/benchexec
+
+SPDX-FileCopyrightText: 2021 Dirk Beyer <https://www.sosy-lab.org>
+SPDX-FileCopyrightText: 2024 Levente Bajczi
+SPDX-FileCopyrightText: Critical Systems Research Group
+SPDX-FileCopyrightText: Budapest University of Technology and Economics <https://www.ftsrg.mit.bme.hu>
+
+SPDX-License-Identifier: Apache-2.0
+-->
+# BenchExec Extension for Benchmarking via SLURM
+
+This Python script extends BenchExec, a benchmarking framework, to facilitate benchmarking via SLURM, optionally using a Singularity container.
+
+In case of problems, contact / tag in an issue: [Levente Bajczi](https://github.com/leventeBajczi)
+
+## Preliminaries
+
+* *SLURM* is an open-source job scheduling and workload management system used primarily in high-performance computing (HPC) environments.
+* *Singularity* is a containerization platform designed for scientific and high-performance computing (HPC) workloads, providing users with a reproducible and portable environment for running applications and workflows.
+
+## Requirements
+
+* Singularity (optional), tested with `singularity-ce version 4.0.1`
+* SLURM, tested with `slurm 22.05.7` on `Red Hat Enterprise Linux 8.6 (Ootpa)`, kernel version `4.18.0-372.9.1.el8.x86_64`
```

Comment:
Do you know anything about likely compatibility? "tested with" is good, but if we can write something like "should work with version 4.0 or newer" in addition, it would be even better..

The versions of RHEL and kernel are likely not so relevant? I would omit them.

Reply:
I don't have a strong enough understanding of SLURM's/singularity's version history to make claims about compatibility, and unfortunately I cannot change the installed versions. I suppose I could add an extra sentence that within the major version there should not be any compatibility problems.

RHEL / kernel versions are indeed not that relevant. Thanks for pointing this out, I'll remove them.